classes/module-signing.bbclass

```diff
@@ -26,8 +26,8 @@ do_sign_modules() {
         [ -z "$SIG_HASH" ] && bbfatal CONFIG_MODULE_SIG_HASH is not set in .config
 
         # ${B} for kernel, ${STAGING_KERNEL_BUILDDIR} for modules
-        for SIGN_FILE in ${STAGING_KERNEL_BUILDDIR}/scripts/sign-file \
-                         ${B}/scripts/sign-file ; do
+        for SIGN_FILE in ${B}/scripts/sign-file \
+                         ${STAGING_KERNEL_BUILDDIR}/scripts/sign-file ; do
```

Comment:
While I'm fine with this, shouldn't this be addressed with a variable to use either the native sysroot for out-of-tree modules and `${B}` for the kernel?
A little like `autotools-brokensep`, except this one would be `module-signing-out-of-tree.bbclass` or something?

Reply:
Yeah, that's probably better.

I could do:
SIGN_FILE = "${STAGING_KERNEL_BUILDDIR}/scripts/sign-file"
in module-signing.bbclass
and then SIGN_FILE = "${B}/scripts/sign-file" in linux-openxt.inc without introducing a new bbclass.